Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -391,10 +403,25 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
       .map { $0 }
 
     self.checkoutError = checkoutCompleteSignal.signal.errors()
+
+    self.processingViewIsHidden = Signal.merge(
+      initialData.mapConst(true),
+      self.submitButtonTappedProperty.signal.mapConst(false),
+      self.applePayButtonTappedSignal.mapConst(false),
+      newPaymentIntentForApplePayError.mapConst(true), // Apple pay payment intent error.
+      validateCheckoutError.mapConst(true), // Card validation error terminates flow.
+      self.checkoutTerminatedProperty.signal.mapConst(true), // Error/cancellation in VC.
+      checkoutCompleteSignal.signal.mapConst(true) // Checkout completed.
+    )
```

Comment:
I see what you mean about worrying about edge cases in here. I don't see any obvious errors - but for readability, you could sort the `merge` into clearer groups of `true` and `false`.

Reply:
Done!